lib/addons/cluster-autoscaler/index.ts

```diff
@@ -42,8 +42,9 @@ const defaultProps: ClusterAutoScalerAddOnProps = {
  * Version of the autoscaler, controls the image tag
  */
 const versionMap = new Map([
-    [KubernetesVersion.of("1.25"), "9.25.0"],
-    [KubernetesVersion.V1_25, "9.25.0"],
+    [KubernetesVersion.of("1.26"), "9.29.0"],
+    [KubernetesVersion.V1_26, "9.29.0"],
+    [KubernetesVersion.V1_25, "9.29.0"],
     [KubernetesVersion.V1_23, "9.21.0"],
```

Comment:
There's missing one for V1.24, which should be mapped to "9.25.0"

Reply:
mapping for V1.24 is added back